src/github_runner_manager/types_/__init__.py

```diff
@@ -96,3 +96,15 @@ class RepoPolicyComplianceConfig(BaseModel):
 
     token: str
     url: AnyHttpUrl
+
+
+class SystemUserConfig(BaseModel):
+    """Configuration for which user to use when spawning processes or accessing resources.
+
+    Attributes:
+        user: The user to choose when spawning processes or accessing resources.
+        group: The group to choose when spawning processes or accessing resources.
+    """
+
+    user: str
+    group: str
```

Comment:
The idea is to let the charm create the user (I think this is charms responsibility to prepare the environment accordingly) and pass the user in a config.